src/lambda/wizards/samDeployWizard.ts

```diff
@@ -299,35 +299,83 @@ export class DefaultSamDeployWizardContext implements SamDeployWizardContext {
     /**
      * Retrieves an S3 Bucket to deploy to from the user.
      *
+     * @param selectedRegion Selected region for S3 client usage
```

Comment:
Since this function is a retrofit, it might be a good time to add testing coverage to it (it seems pretty standalone based on this PR but I might be overlooking something). In case it helps, another recent PR has a [pretty good way to stub the picker](https://github.com/aws/aws-toolkit-vscode/pull/1288/files#diff-f8c099d41c88a2251ca35094fcb582bcR65)

Reply:
I feel like none of our wizards have any testing on the step level, period. Might be worth investigating now that we have mocks for the selectors.